src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectEditingPreferencesPane.java

```diff
@@ -15,13 +15,12 @@
 package org.rstudio.studio.client.projects.ui.prefs;
 
 import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.ui.Label;
 import org.rstudio.core.client.ElementIds;
 import org.rstudio.core.client.prefs.PreferencesDialogBaseResources;
 import org.rstudio.core.client.prefs.RestartRequirement;
 import org.rstudio.core.client.resources.ImageResource2x;
-import org.rstudio.core.client.widget.NumericValueWidget;
-import org.rstudio.core.client.widget.OperationWithInput;
-import org.rstudio.core.client.widget.TextBoxWithButton;
+import org.rstudio.core.client.widget.*;
```

Comment:
Oops -- this is CLion being a little overzealous with the automatic imports, I can narrow this down